aggregator/src/bin/janus_cli.rs

```diff
@@ -167,27 +188,47 @@ async fn provision_tasks<C: Clock>(datastore: &Datastore<C>, tasks_file: &Path)
     };
 
     // Write all tasks requested.
-    let tasks = Arc::new(tasks);
     info!(task_count = %tasks.len(), "Writing tasks");
-    datastore
+    let written_tasks = datastore
         .run_tx(|tx| {
-            let tasks = Arc::clone(&tasks);
+            let tasks = tasks.clone();
             Box::pin(async move {
-                for task in tasks.iter() {
+                let mut written_tasks = Vec::new();
+                for mut serialized_task in tasks.into_iter() {
+                    // Check whether any HPKE configs already exist for this task so we can ensure
```

Comment:
We delete the existing task before creating a new one -- why do we need to care about collisions?

Reply:
...you're right, we don't. I really wish I had thought of that before writing the code to check for and handle collisions. Oh well!